src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
```

Comment:
apply BEM class names recommendations for block elements
```suggestion
      <div class="stars__star"></div>
```